app/src/main/java/org/oppia/android/app/onboarding/CreateProfileActivityPresenter.kt

```diff
@@ -1,12 +1,20 @@
 package org.oppia.android.app.onboarding
 
+import android.os.Bundle
+import android.view.Gravity.apply
```

Comment:
Unintentional import?

Reply:
It unfortunately doesn't catch certain types of imports. Might be something they've fixed (since we're on a very old version of ktlint), but I'd rather move over to ktfmt, anyway. :)